packages/extension/src/ui/features/actions/TxHashContainer.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from "react"
+import styled from "styled-components"
+import { CopyTooltip } from "../../components/CopyTooltip"
+
+const TxHash = styled.div`
+  margin-top: 1px;
+  background: ${({ theme }) => theme.neutrals800};
+  border: 1px solid ${({ theme }) => theme.bg2};
+  color: ${({ theme }) => theme.text4};
+  padding: 9px 13px 8px;
+  overflow-wrap: break-word;
+  font-size: 14px;
+  line-height: 120%;
+  border-radius: 5px;
+`
+
+export const TxHashContainer: FC<{ txId: string }> = ({ txId }) => {
+  return (
+    <CopyTooltip copyValue={txId} message="Copied">
+      <TxHash>{`TxHash: ${splitTxHash(txId)}`}</TxHash>
+    </CopyTooltip>
+  )
+}
+
+// write a function that split a txHash into segments with each segment having 16 character
```

Comment:
I guess this is written already? :)

Reply:
I moved this function from [here](https://github.com/alephium/extension-wallet/pull/137/files#diff-687d46a84f5755579686abedf84eacdde281140427d864578e9ca50fe9e7f2e8L296), and I thought this was a code comment, do you think we should keep it?

Reply:
Ok, looks like a todo that we can remove.